### DISCUSSION TITLE
Configure One-Way Bucket Replication Using the Environment Variables.

### DISCUSSION THREAD

**Discussion by namegorm** [minio/minio#17999] [Q&A] [answered]
Hello.

I found two sections in the official documentation:

1) Configure One-Way Bucket Replication Using the MinIO Console

2) Configure One-Way Bucket Replication Using the Command Line

What if I only want to use environment variables to configure replication. Is it possible?

**Comment by harshavardhana** [comment, marked as answer]
No
